src/nomad_analysis/nomad_plugin.yaml

```diff
@@ -1,4 +1,4 @@
 plugin_type: schema
 name: schemas/nomad_analysis
-description: Analyses parsed data files |
-  This plugin is used to analyze parsed data files.
\ No newline at end of file
+description: |
+  This plugin is used to analyze parsed raw data for spectral profiles in the standard NOMAD schema.
```

Comment:
Should we not keep the description more general? As I understand, spectral profiles will one part (perhaps a big and important one) of the whole idea of analysis

Reply:
Agree. I changed it to:

  This plugin is used to analyze parsed raw data from the NOMAD standard schema.